### PR TITLE
Enable spyglass

### PR DESCRIPTION
**Pull request by fejta**
ref https://github.com/istio/test-infra/issues/1396 https://github.com/istio/test-infra/issues/1239

/assign @utka @howardjohn 

Based off https://github.com/kubernetes/test-infra/pull/11302 

Following https://github.com/kubernetes/test-infra/tree/master/prow/spyglass#config

### CODE REVIEW

**Comment by howardjohn** [comment]
https://prow.istio.io/view/gcs/istio-prow/pr-logs/directory/integ-telemetry-k8s-presubmit-tests-master/1515 gives a 404,
https://prow.k8s.io/view/gcs/istio-prow/pr-logs/directory/integ-telemetry-k8s-presubmit-tests-master/1515
does not right now. 

Is this expected?

**Comment by fejta** [comment]
/hold

Let me investigate...

**Comment by fejta** [comment]
Apparently the issue is I cannot follow instructions: https://github.com/kubernetes/test-infra/tree/master/prow/spyglass#config

**Comment by fejta** [comment]
/hold cancel

Let's first turn on spyglass and prove it works, then enable it by default after that

File: prow/config.yaml [hunk 1]
```diff
@@ -26,6 +26,22 @@ sinker:
   max_pod_age: 6h
 
 deck:
+  spyglass:
```
Comment (fejta) [inline prow/config.yaml:29]: Will address the plank todo above (which makes the github status link go to spyglass) once we validate this is working

**Review by fejta — COMMENTED** [review]
/cc @cjwagner

**Review by cjwagner — APPROVED** [review]

**Comment by cjwagner** [comment]
LGTM

**Review by howardjohn — APPROVED** [review]
/lgtm

**Comment by fejta** [comment]
@cjwagner can you apply this?

**Comment by cjwagner** [comment]
Done. Seems to be working now:
https://prow.istio.io/view/gcs/istio-prow/pr-logs/directory/integ-telemetry-k8s-presubmit-tests-master/1515

**Comment by fejta** [comment]
Thanks and great!

**Pull request merged**